ethash/src/lib.rs

```diff
@@ -135,7 +138,9 @@ impl EthashManager {
 
 #[test]
 fn test_lru() {
-	let ethash = EthashManager::new(&::std::env::temp_dir(), None);
+	use tempdir::TempDir;
+
+	let ethash = EthashManager::new(TempDir::new("").unwrap().path(), None);
```

Comment:
It won't work as expected, cause `TempDir` is dropped before `EthashManager`, so the directory won't be cleaned up